src/antares/service/base_services.py

```diff
@@ -471,6 +471,13 @@ def delete(self, children: bool) -> None:
         """
         pass
 
+    @abstractmethod
+    def read_areas(self) -> List:
```

Comment:
You can type that it's specifically a List of areas: List[Area]

Reply:
even better with `list[Area]` to avoid having to import `List` from typing which is technically not the correct choice anymore since `list` got support for `[ ]` .